students/jraising/lesson03/mailroom1.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct 31 23:19:05 2019
+
+@author: jraising
+"""
+
+donors = {'William Gates, III': [653784.49, 1000.50], 
+          'Mark Z': [16396.10, 30000, 20000.30],
+          'Jeff B': [877.33],
+          'Paul Allen': [708.42, 780, 444.20],
+          'Satya':[ 1000, 500.33]
+          }
+         
+
+def menu_selection(prompt, disp_dir):
+    while True:
+        response = input(prompt)
+        if disp_dir[response]() == "Exit menu":
+            break
+ 
+def thankYou():
+    fullName = input("please enter the full name: ")
+    if fullName == "list":
```

Comment:
If you enter "list" the user, after printing all donors, need be prompted to enter a name (from the list) and not amount. The way you implemented, "list" is donating the specified money.